jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3Session.java

```diff
@@ -859,6 +857,14 @@ public String toString()
         return String.format("%s@%x[streams=%d,%s]", getClass().getSimpleName(), hashCode(), streamCount.get(), closeState);
     }
 
+    /**
+     * <p>Exception indicating that the HTTP/2 stream was not handled
+     * by the server, and can therefore be retried, if possible.</p>
+     */
+    public static class RetryStreamException extends RuntimeException
```

Comment:
This doesn't need to be a `RuntimeException`, so it could extend from `Exception`.